### DISCUSSION TITLE
iOS

### DISCUSSION THREAD

**Discussion by tomcontr** [ExistentialAudio/BlackHole#158] [Feature Requests]
Is this possible to be ported to iOS as well?

**Comment by devinroth** [comment]
I dont think so. Regardless apple has an api for interapp audio on ios. 

How exactly would you use it?

Devin

**Comment by tomcontr** [comment]
Im already streaming spotify an other audio apps using Icecast in windows
an mac with audio driver like black hole.
It would be nice to be able to do that from an iPad.
